src/main/java/com/imsweb/datagenerator/gui/StandaloneNaaccrDataGenerator.java

```diff
@@ -252,6 +252,18 @@ public void keyReleased(KeyEvent e) {
         optionsPnl.setBorder(new CompoundBorder(border, new EmptyBorder(10, 15, 5, 15)));
         pnl.add(optionsPnl);
 
+        // registry id
+        JPanel registryId = new JPanel();
```

Comment:
I didn't expect you to modify the GUI, but that's great. That's something I will have to test using the app though, can't really tell if it "looks" good or not :-)